tests/support/server.tcl

```diff
@@ -373,6 +373,8 @@ proc run_external_server_test {code overrides} {
 
     r flushall
     r function flush
+    r script flush
+    r config resetstat
```

Comment:
yes, in another branch, but it's probably a good idea to reset the server as much as we can to avoid unexpected issues in external where tests assume things.